subsys/mgmt/mcumgr/transport/smp_bt.c

```diff
@@ -17,10 +17,11 @@
 #include <zephyr/bluetooth/gatt.h>
 
 #include <zephyr/mgmt/mcumgr/smp_bt.h>
-#include <zephyr/mgmt/mcumgr/buf.h>
 
 #include <zephyr/mgmt/mcumgr/smp.h>
+#include <zcbor_common.h>
```

Comment:
No zcbor usage in this file

Reply:
Hm. It really baffles me how come I did add the <zcbor_common.h> to so many files, where it is not needed.